sonic-ycabled/ycable/ycable_utilities/y_cable_table_helper.py

```diff
@@ -10,7 +10,7 @@
 
 MUX_CABLE_STATIC_INFO_TABLE = "MUX_CABLE_STATIC_INFO"
 MUX_CABLE_INFO_TABLE = "MUX_CABLE_INFO"
-TRANSCEIVER_STATUS_TABLE = 'TRANSCEIVER_STATUS'
```

Comment:
What's the difference between two tables? 

Is `TRANSCEIVER_STATUS_TABLE` referred anywhere else?

Reply:
fixed, checked and no other usage except subscriber state